### DISCUSSION TITLE
RoomControllerV2 Climate Control

### DISCUSSION THREAD

**Discussion by northernblues55** [JoDehli/PyLoxone#333] [Q&A]
Hello,
sorry for the question, but should there be a control the set the target temperature?
i can only change the the hlk mode and see the current temperature.
![grafik](https://github.com/user-attachments/assets/4c6682ca-6802-406b-97f5-3f6bdb864d75)
![grafik](https://github.com/user-attachments/assets/5374d9ce-63b5-4673-9c40-a8c75b66929c)

**Comment by JoDehli** [comment]
@northernblues55 it is already discussed why it is not supported by pyloxone. For more infos read the issues about this topic.

**Reply by ppitonak** [reply]
Where was it discussed? I checked discussions and issues (both open and closed) but could not find it :(
